tests/main/remodel-kernel/task.yaml

```diff
@@ -34,6 +38,22 @@ restore: |
         exit
     fi
 
+    # Wait for the system to stop processing stuff
+    # XXX: What was it processing? I got this error:
+    # error: snap "pc-kernel" has "refresh-snap" change in progress
```

Comment:
Probably finishing the last kernel refresh in the `execute:` section

Reply:
I'm not sure, do you mean the code before the last REBOOT case? Perhaps it should just let the system reboot by itself?